lib/online_order.rb

```diff
@@ -0,0 +1,79 @@
+require 'csv'
+require_relative 'order'
+module Grocery
+  class OnlineOrder < Grocery::Order
+    attr_reader :customer, :status
+
+    def initialize (id, products, customer, status = :pending)
+      super(id, products)
```

Comment:
You do a good job of using inheritance and the `super` keyword to keep this class DRY. Good work!